src/blended_cg.jl

```diff
@@ -381,23 +374,22 @@ function minimize_over_convex_hull!(
             if L_reduced / mu_reduced > 1.0
                 new_weights, number_of_steps =
                     accelerated_simplex_gradient_descent_over_probability_simplex(
-                        active_set.weights,
+                        active_set,
```

Comment:
the argument `active_set` is now passed twice to accelerated_simplex_gradient_descent_over_probability_simplex